spaceship/spec/provisioning_profile_spec.rb

```diff
@@ -205,52 +205,54 @@
   end
 
   describe "#repair" do
-    let(:profile) { Spaceship::ProvisioningProfile.all.first }
+    let(:profile) { Spaceship::ProvisioningProfile.all.detect { |pp| pp.id == 'PP00000006' } }
 
     it "repairs an existing profile with added devices" do
       profile.devices = Spaceship::Device.all_for_profile_type(profile.type)
-      expect(client).to receive(:repair_provisioning_profile!).with('2MAY7NPHRU', 'net.sunapps.7 AppStore', 'store', '572XTN75U2', [cert_id], ["AAAAAAAAAA", "BBBBBBBBBB", "CCCCCCCCCC", "DDDDDDDDDD"], mac: false, sub_platform: nil).and_return({})
+      expect(client).to receive(:repair_provisioning_profile!).with('PP00000006', 'delete.me.please AppStore', 'store', '2UMR2S6P4L', [cert_id], ["AAAAAAAAAA", "BBBBBBBBBB", "CCCCCCCCCC", "DDDDDDDDDD"], mac: false, sub_platform: nil).and_return({})
```

Comment:
aren't you missing some test data?

Where do you have the pp with id `PP00000006`, etc. It is needed to be returned by `Spaceship::ProvisioningProfile.all.` right ?